### ISSUE TITLE
Pin preload hook throws an exception

### ISSUE THREAD

**Issue by o1y** [jacksleight/statamic-bard-texstyle#66]
## Bug Description
When using Bard Texstyle Pro on a fresh Statamic 5 installation the control panel just throws a `json_decode()` exception. This happens because `$this->field->value()` returns an array instead of a string in Bard's preload hook [/src/Nodes/Pin.php:42](https://github.com/jacksleight/statamic-bard-texstyle/blob/main/src/Nodes/Pin.php#L42).

## How to Reproduce
1. Install Statamic 5 through Statamic/CLI
2. Add Bard Field to Blueprint
3. `composer require jacksleight/statamic-bard-texstyle`
4. Enable statamic-bard-texstyle Pro Edition
5. Go to the entry's Publish form

## Extra Detail
```
[2024-05-12 11:41:18] local.ERROR: json_decode(): Argument #1 ($json) must be of type string, array given {"userId":"6500ba0d-0bc3-4219-9b2c-3ba099ea58ef","exception":"[object] (TypeError(code: 0): json_decode(): Argument #1 ($json) must be of type string, array given at /.../bard-texstyle-debug/vendor/jacksleight/statamic-bard-texstyle/src/Nodes/Pin.php:42)
```

## Environment
```
Environment
Application Name: Statamic
Laravel Version: 11.7.0
PHP Version: 8.3.6
Composer Version: 2.7.6
Environment: local
Debug Mode: ENABLED
URL: bard-texstyle-debug.test
Maintenance Mode: OFF

Cache
Config: NOT CACHED
Events: NOT CACHED
Routes: NOT CACHED
Views: CACHED

Drivers
Broadcasting: log
Cache: statamic
Database: sqlite
Logs: stack / single
Mail: log
Queue: sync
Session: file

Statamic
Addons: 1
Sites: 1
Stache Watcher: Enabled
Static Caching: Disabled
Version: 5.0.2 Solo

Statamic Addons
jacksleight/statamic-bard-texstyle: 3.2.1
```

**Comment by jacksleight** [comment]
Whoops! Given that I'm the one who PR'd the change that causes this you'd think I'd have remembered! 🤦‍♂️

Thanks for reporting, will fix ASAP.

**Comment by jacksleight** [comment]
Fix in 3.2.2
